lib/ImageHelper.php

```diff
@@ -367,7 +367,10 @@ private static function analyze_url( $url ) {
 			}
 		} else {
 			if ( !$result['absolute'] ) {
```

Comment:
Right now `$tmp` only gets a value when `$post !== false`, should this line be brought back?